index.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  "rules": {
+    "at-rule-empty-line-before": [ 2, "always", {
+      except: ["blockless-group"],
+    } ],
+    "at-rule-no-vendor-prefix": 0,
```

Comment:
Any reason why you're explicitly turning these rules off? Every rule in `stylelint` is off by default, so the only time you need to manually turn off a rule is if you're extending another config.

Also, see the note about the pipeline and how you might be able to turn this rule on.